### PR TITLE
Add prepublish text length guidelines

### LINKED ISSUE #4914: Pre-Publish Checklist: Text guidelines (guidance) Logic
## Feature Description

- Too much text on screen (>200 characters), suggest a page attachment, breaking up the text into multiple screens, or keeping the total number of pages with a lot of text to less than 10% of the pages in the story.
- Too little text: entire story contains less than 100 characters (per amp linter)

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by zachhale**
## Summary

Adds prepublish text length checks

[ticket](https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/4914)

## Relevant Technical Choices

Return objects are aligned with #5029, but actual use scenario is fuzzy.

## To-do

* [x] Update to use types and enums defined in #5029

## Testing Instructions

---

Fixes #4914

### CODE REVIEW

File: assets/src/edit-story/app/prepublish/text.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  });
+  return characterCount;
+};
+
+export const pageTooMuchText = (page) => {
+  const characterCount = characterCountForPage(page);
+
+  if (characterCount > maxPageCharacterCount) {
+    return {
+      message: 'Too much text on page',
```
Comment (embarks) [inline assets/src/edit-story/app/prepublish/text.js]: I forgot we should translate the text :) see [here](https://github.com/google/web-stories-wp/pull/5029/files#diff-5ea3e91693715472e1edeff0dafe175ca48e6b0f0b72153d8a89a189a4cd4133R20)

Comment (zachhale) [reply assets/src/edit-story/app/prepublish/text.js]: Thanks! I was going to bring that up today! Is there a doc with finalized text for these guidelines?

Comment (embarks) [reply assets/src/edit-story/app/prepublish/text.js]: I don't think so. All I've seen is the [product brief](https://docs.google.com/document/d/1GJZljicdXcP7oRdywsF3EGUaCVpCHsDrK2gUMubnrbk/edit?pli=1#) and the tickets themselves

**Comment by zachhale** [comment]
@googlebot I signed it!

File: assets/src/edit-story/app/prepublish/text.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+ * @return {Object} Prepublish check response
+ */
+export function pageTooMuchText(page) {
+  const characterCount = characterCountForPage(page);
+
+  if (characterCount > MAX_PAGE_CHARACTER_COUNT) {
+    return {
+      message: __('Too much text on page', 'web-stories'),
+      pageId: page.id,
+      type: 'guidance',
```
Comment (swissspidy) [inline assets/src/edit-story/app/prepublish/text.js]: We probably should have an enum somewhere. Something like

```
const PRE_PUBLISH_MESSAGE_TYPES = {
  GUIDANCE: 'guidance',
  // ...
}
```

Comment (embarks) [reply assets/src/edit-story/app/prepublish/text.js]: Hey Zach, I added some typedefs and this enum on my PR. What do you think about these changes?
https://github.com/google/web-stories-wp/pull/5029/commits/695cc783e1a9ef5e632569ef7c48e21c9bd59026

Comment (zachhale) [reply assets/src/edit-story/app/prepublish/text.js]: @embarks that all looks great to me. I'll adopt those types and enum after your merge.

**Review by embarks — APPROVED** [review]
✨

**Pull request merged**